model/ddl.go

```diff
@@ -204,6 +204,16 @@ func (job *Job) IsCancelled() bool {
 	return job.State == JobCancelled || job.State == JobRollbackDone
 }
 
+// IsRollbacking returns whether the job is rollbacking or not.
+func (job *Job) IsRollbacking() bool {
+	return job.State == JobRollback
```

Comment:
s/JobRollback/JobStateRollingback/

Reply:
I will modify the prefix in the next PR.